src/main/java/com/laba/solvd/bankhierarchy/Main.java

```diff
@@ -0,0 +1,150 @@
+package com.laba.solvd.bankhierarchy;
+
+import com.laba.solvd.bankhierarchy.bankingcore.ATM;
+import com.laba.solvd.bankhierarchy.bankingcore.Bank;
+import com.laba.solvd.bankhierarchy.bankingcore.Branch;
+import com.laba.solvd.bankhierarchy.exceptions.CustomerAlreadyExistsException;
+import com.laba.solvd.bankhierarchy.exceptions.DuplicateAtmException;
+import com.laba.solvd.bankhierarchy.exceptions.InsufficientFundsException;
+import com.laba.solvd.bankhierarchy.exceptions.InvalidCustomerException;
+import com.laba.solvd.bankhierarchy.financial.Account;
+import com.laba.solvd.bankhierarchy.financial.Card;
+import com.laba.solvd.bankhierarchy.people.Customer;
+import com.laba.solvd.bankhierarchy.people.Employee;
+import com.laba.solvd.bankhierarchy.people.Position;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Main {
+
+    private static final Logger LOGGER = LogManager.getLogger(Main.class);
+
+    public static void main(String[] args) {
+
+        // Creating a bank
```

Comment:
please don't start and end your methods with an empty line, check all methods and classes and fix it
```suggestion
    public static void main(String[] args) {
        // Creating a bank
```